lib/resourceapply/machineconfig.go

```diff
@@ -80,21 +81,32 @@ func ApplyMachineConfigNode(client mcfgclientalphav1.MachineConfigNodesGetter, r
 
 // ApplyControllerConfig applies the required machineconfig to the cluster.
 func ApplyControllerConfig(client mcfgclientv1.ControllerConfigsGetter, required *mcfgv1.ControllerConfig) (*mcfgv1.ControllerConfig, bool, error) {
+	klog.Info("Getting existing ControllerConfig with name: ", required.GetName())
 	existing, err := client.ControllerConfigs().Get(context.TODO(), required.GetName(), metav1.GetOptions{})
 	if apierrors.IsNotFound(err) {
+		klog.Infof("ControllerConfig not found, creating new one")
```

Comment:
suggestion (non-blocking): By comparison to the above, since you're not using a formatting string, use `klog.Info()` here instead.